src/lib/libhtml5_webgl.js

```diff
@@ -109,7 +109,7 @@ var LibraryHtml5WebGL = {
 #endif
 
     var canvas = findCanvasEventTarget(target);
-
+    if (canvas.canvas) { canvas = canvas.canvas; }
```

Comment:
Looks like you canvas can be null here?   How about `canvas?.canvas`?

Reply:
Yes, I think the condition in the if also must be guarded. `if (canvas && canvas.canvas) { canvas = canvas.canvas }`?